sips/mev.md

```diff
@@ -0,0 +1,111 @@
+
+> Note: producing a `BlindedBeaconBlock` before and shortly thereafter (delay depends on the builder) the registration may result in either:
+> 1. Beacon node falls back to locally-built block from it's execution layer.
+> 2. Builder doesn't reward the validator's `fee_recipient` because it isn't aware of it.
+> 3. Builder rewards a potentially different `fee_recipient` from the validator's latest registration (such as a registration prior to onboarding to SSV.)
+
+#### Issues
+
+##### Gas limit
+
+Within the proposed construction, the gas limit is set by validators only and relies on them to update it if needed.
+
+If gas limit is likely to change to adjust to network conditions, then this isn't ideal because validators would have to track of it, and when it changes — produce a new signature and publish an update transaction to the contract (which may be very expensive when the network is congested).
+
+> TODO: examine the code of validator clients such as Lighthouse and Prysm to figure this out.
+
+### Blinded block proposals
+
+#### Scenario A — Block is published only by operators who share builder with leader
+
+1. Leader produces a `BlindedBeaconBlock` from their Beacon node at [/eth/v1/validator/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock) and broadcasts it.
+
+2. Operators sign the `BlindedBeaconBlock`.
+
+3. Operators publish the `SignedBlindedBeaconBlock` to their Beacon node at [/eth/v1/beacon/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Beacon/publishBlindedBlock).
+  
+    The Beacon node reveals the signature to the builder, which (ideally) reveals the transactions in return, after which it unblinds the block and publishes it to the network.
+
+    ##### Issues
+
+    Unrealistic?
+
+#### Secnario B — Block is published by all operators
+
+1. Leader produces a `BlindedBeaconBlock` from their Beacon node at [/eth/v1/validator/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock) and broadcasts it together with the endpoint of it's builder as `BuilderBlindedBeaconBlock`.
+
+    ```go
+    type BuilderBlindedBeaconBlock {
+        BuilderEndpoint    string
+        BlindedBeaconBlock *spec.VersionedBlindedBeaconBlock
+    }
+    ```
+
+2. Operators sign the `BlindedBeaconBlock`.
+
+3. Operators submit the `SignedBlindedBeaconBlock` to the builder's endpoint at [/eth/v1/builder/blinded_blocks](https://ethereum.github.io/builder-specs/#/Builder/submitBlindedBlock) to reveal the transactions and unblind the block into a `SignedBlindedBlock`.
+
+4. Operators publish the `SignedBeaconBlock` to their Beacon node at [/eth/v1/beacon/blocks](https://ethereum.github.io/beacon-APIs/#/Beacon/publishBlock).
+
+    ##### Issues
+
+    ###### Unknown builder
```

Comment:
This means that blinded blocks are basically only broadcastable by the leader...